logging/access.go

```diff
@@ -0,0 +1,126 @@
+package logging
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"net"
+	"net/http"
+	"time"
+)
+
+const (
+	dateFormat      = "02/Jan/2006:15:04:05 -0700"
```

Comment:
that's how go date format works. i agree that it is strange. this particular format is from the apache access log. we didn't test whether this format works with the go date fmt magic, but we saw it used in another library doing the same. (no, the code of that library was not to be reused.)